### PR TITLE
IDEM-168

### PR DESCRIPTION
**Pull request by JasminFarhi**

### CODE REVIEW

File: app/src/components/UserDetailsHeader.tsx [hunk 1]
```diff
@@ -45,7 +46,10 @@ const UserDetailsHeader: React.FC = () => {
       const { fingerPrint } = key;
 
       const lastEightChar =
-        fingerPrint?.slice(fingerPrint.length - 8) ?? "Missing PGP Fingerprint";
+        fingerPrint
+          .split(":")
+          .join(" ")
+          .slice(fingerPrint.length - 10) ?? "Missing PGP Fingerprint";
```
Comment (pondpiu) [inline app/src/components/UserDetailsHeader.tsx:52]: variable name lastEightChar slice last 10 char

File: app/src/navigation/screens/PGPScreen.tsx [hunk 2]
```diff
@@ -80,22 +72,16 @@ const PGPScreen: React.FC = () => {
         }`
       );
       console.error(error);
+      console.log(generateAndPublishNewPgpKey);
```
Comment (pondpiu) [inline app/src/navigation/screens/PGPScreen.tsx:75]: log

File: app/src/navigation/screens/PGPScreen.tsx [hunk 3]
```diff
@@ -150,7 +136,7 @@ const PGPScreen: React.FC = () => {
           <View style={styles.button}>
             <Button
               title={"Import my Private Key"}
-              onPress={() => importMyPrivateKey(keyText as string)}
+              onPress={() => keyText as string}
```
Comment (pondpiu) [inline app/src/navigation/screens/PGPScreen.tsx:139]: this do nothing

File: app/src/components/UserDetailsHeader.tsx [hunk 4]
```diff
@@ -15,6 +15,7 @@ import useMnemonic from "../hooks/useMnemonic";
 import { pgpLocalStorage } from "../utils/local-storage";
 import { useEffect, useState } from "react";
 import { ClaimTypeConstants } from "../constants/common";
+import { splitSignature } from "ethers/lib/utils";
```
Comment (pondpiu) [inline app/src/components/UserDetailsHeader.tsx:18]: unused import

**Review by pondpiu — CHANGES_REQUESTED** [review]

File: app/src/components/UserDetailsHeader.tsx (hunk 4 above)
Comment (JasminFarhi) [reply app/src/components/UserDetailsHeader.tsx:18]: removed :)

File: app/src/navigation/screens/PGPScreen.tsx (hunk 2 above)
Comment (JasminFarhi) [reply app/src/navigation/screens/PGPScreen.tsx:75]: removed

File: app/src/components/UserDetailsHeader.tsx (hunk 1 above)
Comment (JasminFarhi) [reply app/src/components/UserDetailsHeader.tsx:52]: Updated name to lastEightCharAndSpaces

File: app/src/navigation/screens/PGPScreen.tsx (hunk 3 above)
Comment (JasminFarhi) [reply app/src/navigation/screens/PGPScreen.tsx:139]: fixed this up also

**Review by xiangzhe0812 — APPROVED** [review]

**Review by xiangzhe0812 — DISMISSED** [review]

File: app/src/hooks/usePpg.ts [hunk 5]
```diff
@@ -11,7 +11,7 @@ import useApi from "./useApi";
 type Hooks = {
   generateKeyPair: (
     name: string | undefined,
-    email: string | undefined
+    email: string | undefined
```
Comment (bitcoinbrisbane) [inline app/src/hooks/usePpg.ts:14]: let roll this back

**Review by bitcoinbrisbane — CHANGES_REQUESTED** [review]
No need to commit usePGP.ts

**Review by bitcoinbrisbane — APPROVED** [review]

**Pull request merged**